daemons/fenced/pacemaker-fenced.c

```diff
@@ -53,6 +53,9 @@ static gboolean stonith_shutdown_flag = FALSE;
 static qb_ipcs_service_t *ipcs = NULL;
 static xmlNode *local_cib = NULL;
 static pe_working_set_t *fenced_data_set = NULL;
+static const unsigned long long data_set_flags = pe_flag_quick_location
```

Comment:
I don't think there's any real difference on x86, but it'd be more consistent across platforms to use something like uint64_t for this.  We're already using it for other flags elsewhere.

Reply:
Yep but this is used as pe_working_set_t:flags which is public API, so it will have to be changed at a compatibility break